docs/release.sh

```diff
@@ -116,17 +84,20 @@ invalidate_cache() {
 
 	aws configure set preview.cloudfront true
 
-	files=$(find site/ -not -name "*.md*" -type f | sed 's/site\///g')
+	# Get all the files
+	# not .md~ files
+	# not files w spaces bc they mess up the array
```

Comment:
I'd personally go for something like:
```bash
local IFS=$'\n'
files=( $(find ....) )
unset IFS
```

IMO, people who actually put newlines in filenames are _asking_ for trouble